build/configs/rtl8730e/flat_dev/defconfig

```diff
@@ -1231,6 +1231,8 @@ CONFIG_LIBCXX_HAVE_LIBSUPCXX=y
 # CONFIG_AVS_DEVICE_SDK is not set
 # CONFIG_AWS_SDK is not set
 # CONFIG_NETUTILS_CODECS is not set
+CONFIG_EXTERNAL_CMSIS_NN=y
+CONFIG_CMSIS_LIB_DIRECTORY_NAME="rtl8730e"
```

Comment:
CONFIG_ARCH_BOARD has the same value. It shows a board name. RTL8730E has "rtl8730e" and RTL8721CSM has "rtl8721csm" which are the same folder name in build/configs.
How about using it instead of adding new config?